.github/workflows/run-tests.yml

```diff
@@ -45,8 +45,19 @@ jobs:
       - name: Checkout neofs-node repository
         uses: actions/checkout@v3
         with:
-          repository: ${{ github.repository }}
-          ref: ${{ github.event.inputs.neofs_node_ref || github.head_ref || github.ref_name }}
```

Comment:
It's too complicated, why don't you just use the
```
      - name: Checkout neofs-node repository
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
```
It should wok for both PRs created from forks and from neofs-node.
Do not specify `repository`, do no specify `ref`, the action will do all this magic for you as you described in the commit message. Or not?

Reply:
@AnnaShaleva 
`It should wok for both PRs created from forks and from neofs-node.
`
No, it doesn't work.
It's your fix:
https://github.com/vvarg229/neofs-node/blob/master/.github/workflows/run-tests.yml#L50
It's a test PR:
https://github.com/vvarg229/neofs-node/pull/8
It's a result:
https://github.com/vvarg229/neofs-node/actions/runs/5044005505/jobs/9099389504?pr=8
![image](https://github.com/nspcc-dev/neofs-node/assets/126654550/4760131d-ba73-424f-8011-836a7aa4cebf)

Reply:
@vvarg229 
> It's your fix: https://github.com/vvarg229/neofs-node/blob/master/.github/workflows/run-tests.yml#L50

But it's not my fix. Are you sure that you've tested without `repository` and `ref` fields set?

Reply:
> Are you sure that you've tested without `repository` and `ref` fields set?

Yes, I tested it.
For proof, here is the result:
https://github.com/vvarg229/neofs-node/blob/master/.github/workflows/run-tests.yml#L48
https://github.com/vvarg229/neofs-node/actions/runs/5044005505/jobs/9104414914
![image](https://github.com/nspcc-dev/neofs-node/assets/126654550/4d19f2e6-ac59-4aa4-a34f-8e1128ef2b58)

Reply:
I think what @AnnaShaleva meant was
```
--- a/.github/workflows/run-tests.yml
+++ b/.github/workflows/run-tests.yml
@@ -45,8 +45,6 @@ jobs:
       - name: Checkout neofs-node repository
         uses: actions/checkout@v3
         with:
-          repository: ${{ github.repository }}
-          ref: ${{ github.event.inputs.neofs_node_ref || github.head_ref || github.ref_name }}
           path: neofs-node
 
 ################################################################
```
Have you tried it this way?

Reply:
Exactly. It should work properly, because it works in neo-go.